api/managementpb/actions.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+
+package management;
```

Comment:
So it will be part of management API? I have been thinking that we decided it will be separate API package.

Reply:
First I've made `actionspb` directory and actions service, then I realized that it will be something like `/v0/actions/Actions/RunAction` and it seems looks ugly isn't it? So I have decided to move it to the management API in the first iteration. I think it's ok, but I'm not sure so we can discuss it. Also, I don't really like our implementation of management API in `pmm-managed` where we have a lot of services in one package (in one `namespace` in fact). Maybe we should separate those services, or it should be one service implements all functions.

Reply:
yeah, doesn't look good.